scripts/build-package.sh

```diff
@@ -6,7 +6,7 @@ set -e
 echo "building package: ${1}"
 
 pushd ./packages/${1}
-    npm ci
+    npm ci -f
```

Comment:
what does that do?

Reply:
it does a force install, I kept getting flaky github errors around `npm ERR! code EEXIST`. We can fix this before we merge back to main, this is just what fixed it and unblocked me for building the 2 sdks on pr

Reply:
okay maybe leave a comment there so it's obvious, then i'm fine with it for now